armi/operators/tests/test_operatorSnapshots.py

```diff
@@ -61,6 +61,34 @@ def test_mainOperate(self):
         self.o._mainOperate()
         self.assertEqual(self.r.core.p.power, 100000000.0)
 
+    def test_createInterfaces(self):
```

Comment:
Love the test additions to get coverage for the disabled interfaces. 

Though it's not entirely clear to me how these assertions are covering the disabled interfaces? I very well may be missing something. 

Could we simplify and make into one test if we did something like below after a call to `self.o.createInterfaces()`? 
```python
disabledInterfaces = self.o.disabledInterfaces
for interface in self.o.interfaces:
    if i.name in disabledInterfaces or i.function in disabledInterfaces:
        self.assertFalse(i.enabled())
    else:
        self.assertTrue(i.enabled()) 
```

Reply:
If you scroll down a few more lines, you will see the test for disabled interfaces.